.markdownlint.yml

```diff
@@ -1,16 +1,10 @@
-# Line length
-MD013: false
-# Multiple headings with the same content
-MD024: false
+MD013: false # Line length
+MD024: false # Multiple headings with the same content
+MD046: false # indented / fenced code blocks (false positives)
 # These rules should be enabled/disabled eventually
-MD026: false
-MD030: false
-MD031: false
-MD033: false
+MD033:
```

Comment:
Can you add a comment on what this represents?

Reply:
Resolved in 1c941a5.